cmd/level-1/command.go

```diff
@@ -53,7 +53,7 @@ func NewCmd() *cobra.Command {
 		"filter by the Package names which are like *package-name*. Required for Operator Clean-up")
 	cmd.Flags().StringVar(&flags.BundleName, "bundle-name", "",
 		"filter by the Bundle names which are like *bundle-name*")
-	cmd.Flags().StringVar(&flags.FilterBundle, "bundle-image", "",
+	cmd.Flags().StringVar(&flags.BundleImage, "bundle-image", "",
```

Comment:
This change is not related to the tool renaming. At doesn't appear to be. That's fine for now but would be nice to have it in a separate commit but possibly on the same PR.

Reply:
Apologies for changing flags in this PR. I will keep in mind to have a separate PR for each change moving forward

Reply:
Done. Removed unrelated changes from this PR.